pyam/_aggregate.py

```diff
@@ -107,7 +114,10 @@ def _aggregate_region(
     else:
```

Comment:
Please add the following statement to the `if` just above (before computing `_data`):
```
if drop_negative is False:
    raise ValueError("Dropping negative weights can only be used with `weights`!)
```